kaitai_utils.py

```diff
@@ -90,10 +93,13 @@ def _object_to_dict(self, instance: Any, path: str) -> Generator[Dict[str, Any],
                 elif _is_list(value_object):
                     yield _to_lower_camel_case(key), self._list_to_dict(value_object, path)
                 elif isinstance(value_object, bytes):
-                    if len(value_object) < hansken_extraction_plugin.runtime.constants.MAX_CHUNK_SIZE:
-                        child_builder = self.trace.child_builder(path)
-                        child_builder.update(data={'raw': value_object}).build()
-                    yield _to_lower_camel_case(key), "data block of size: " + str(len(value_object))
+                    if len(value_object) > self.max_byte_array_length:
+                        yield _to_lower_camel_case(key), "data block of size: " + str(len(value_object))
+                        if len(value_object) < hansken_extraction_plugin.runtime.constants.MAX_CHUNK_SIZE:
+                            child_builder = self.trace.child_builder(path)
+                            child_builder.update(data={'raw': value_object}).build()
+                    else:
+                        yield _to_lower_camel_case(key), value_object.hex()
```

Comment:
Is this hex value formatted? If not, perhaps that would be nice to do so as well.

Some forms I can think of:
```
00 00 00 00 00 00 00 00
00 00 00 00 00 00 00 00
00 00 00 00 00 00 00 00

# or
00000000 00000000
00000000 00000000
00000000 00000000

# or more like a hex viewer (fake example)
%PDF-1.5.%.x.5  25504446 25504446
```

Not sure if we want to differentiate between small byte arrays (<=16 bytes) or larger arrays?

Any suggestions @Schramp ?

Reply:
Formatting it as a hex viewer might make it annoying to copy the data to a proper hex viewer. The first or second option seem nice to me.